public/js/selfoss-base.js

```diff
@@ -75,7 +90,9 @@ var selfoss = {
             window.setInterval(selfoss.dbOnline.sync, 60*1000);
 
             window.setInterval(selfoss.ui.refreshEntryDatetimes, 60*1000);
-        });
+
+            selfoss.ui.showMainUi();
```

Comment:
After you log in it seems to work fine but if you refresh the page, it overflows. You need to have enough tags or items to fill the whole area, or resize the window before reloading the page. I can reproduce it in both Firefox and Chomium.